### DISCUSSION TITLE
Snug Supports

### DISCUSSION THREAD

**Discussion by GrimSinner** [supermerill/SuperSlicer#1910] [Q&A]
Will this feature be a part of SuperSlicer?

**Comment by supermerill** [comment]
Only cosy ones.

**Reply by GrimSinner** [reply]
Cozy supports it is then!  I saw some good reviews on the snug support feature in Pr u s a.  Just wondering, thanks.

**Reply by supermerill** [reply]
PrusaSlicer 2.4 will be merged when out of beta.
